Dream build - All platforms, all repos logically hosted

IanButterworth opened the pull request:
Todo
Releases for:
- [x] LibVorbis - https://github.com/JuliaIO/LibVorbisBuilder
- [x] LAME - https://github.com/JuliaIO/LAMEBuilder
- [x] x264 - https://github.com/JuliaIO/x264Builder
- [x] x265 - https://github.com/JuliaIO/x265Builder
- [x] FreeType - https://github.com/JuliaGraphics/FreeTypeBuilder/pull/4
- [x] LibVPX - https://github.com/JuliaIO/LibVPXBuilder 
- [x] Bzip2 - https://github.com/JuliaPackaging/Yggdrasil/releases/tag/Bzip2-v1.0.6-0
- [x] Zlib - https://github.com/ianshmean/ZlibBuilder/releases/tag/v1.2.11
- [x] Libass https://github.com/JuliaIO/LibassBuilder

IanButterworth committed:
Update .travis.yml

IanButterworth committed:
disable deploy

IanButterworth committed:
release any .tar.gz

IanButterworth committed:
catch products

IanButterworth committed:
remove musl

IanButterworth committed:
deploy settings

IanButterworth committed:
enable master

IanButterworth committed:
Merge branch 'master' into attempt

IanButterworth committed:
Attempt (#2)

IanButterworth committed:
attempt to add libx264rgb

IanButterworth committed:
change to --enable-encoder=libx264rgb

IanButterworth committed:
Merge pull request #3 from ianshmean/attempt

Attempt to add libx264rgb

IanButterworth committed:
trigger travis on master

IanButterworth committed:
enable all encoders etc

IanButterworth committed:
Merge pull request #4 from ianshmean/enable-all

enable all encoders etc

IanButterworth committed:
enable libx264rgb specifically

IanButterworth committed:
hard coded paths for h264

IanButterworth committed:
typo

IanButterworth committed:
fix paths

IanButterworth committed:
Merge branch 'master' of https://github.com/ianshmean/FFMPEGBuilder

IanButterworth committed:
change cflags to abs paths

IanButterworth committed:
update x264 to provide gray8 etc.

IanButterworth committed:
going for the perfect setup

IanButterworth committed:
Merge remote-tracking branch 'upstream/master'

IanButterworth committed:
Merge branch 'master' into dream_build

IanButterworth committed:
update LAMEBuilder release

IanButterworth committed:
update LibVorbisBuilder release

staticfloat commented:
You guys are doing really good work here, I'm really pleased to see this coming together.  It inspires me to keep on chugging toward the next iteration of BinaryBuilder/Yggdrasil/Pkg.

IanButterworth committed:
update readme

IanButterworth committed:
temporarily disable x265 and libvpx

IanButterworth committed:
cross compile handling

IanButterworth committed:
update secure personal token to mine

IanButterworth committed:
try without docs and skip working platforms

IanButterworth committed:
add sysroot

IanButterworth committed:
fix sysroot with staticfloat's big guns

IanButterworth committed:
update FreeTypeBuilder, disable successful platforms

IanButterworth committed:
update

IanButterworth committed:
x265 pkg-config fix

https://bitbucket.org/multicoreware/x265/issues/371/x265-not-found-using-pkg-config

IanButterworth committed:
fix wins

IanButterworth committed:
Update .travis.yml

IanButterworth commented:
All working locally!
